build.xml

```diff
@@ -193,7 +200,7 @@ lib/plexi submodule or add the the command line argument
         <attribute name="Class-Path" value="${tmp.adaptor.classpath}"/>
         <section name="com/google/enterprise/adaptor/filenet/">
           <attribute name="Implementation-Title"
-            value="Google FileNet Adaptor"/>
+            value="Google Search Appliance Connector for FileNet"/>
```

Comment:
I had also changed the descriptions at the top of build.xml, to a plain "Connector for Foo".